--- old/drivers/input/touchscreen/focaltech_touch/focaltech_flash.c	2023-01-10 19:29:02.000000000 +0000
+++ new/drivers/input/touchscreen/focaltech_touch/focaltech_flash.c	2024-06-24 00:37:32.080147596 +0000
@@ -46,15 +46,12 @@
 * Global variable or extern global variabls/functions
 *****************************************************************************/
 u8 fw_file[] = {
-#include FTS_UPGRADE_FW_FILE
 };

 u8 fw_file2[] = {
-#include FTS_UPGRADE_FW2_FILE
 };

 u8 fw_file3[] = {
-#include FTS_UPGRADE_FW3_FILE
 };

 struct upgrade_module module_list[] = {
